DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace M6Web\Bundle\StatsdTagsBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * To learn more see
+ * {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('m6_statsd_tags');
+
+        $this->addServersSection($rootNode);
+        $this->addClientsSection($rootNode);
+        $this->addDefaultEventSection($rootNode);
+
+        return $treeBuilder;
+    }
+
+    private function addServersSection($rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('servers')
+                    ->useAttributeAsKey('alias', false)
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('address')
+                                ->isRequired()
+                                ->validate()
+                                    ->ifTrue(function ($v) {return substr($v, 0, 6) !== 'udp://'; })
+                                    ->thenInvalid("address parameter should begin with 'udp://'")
+                                ->end()
+                            ->end()
+                            ->scalarNode('port')->isRequired()->end()
+                        ->end()
+                    ->end()
+                ->end()
+            ->end();
+    }
+
+    private function addClientsSection($rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('clients')
+                    ->useAttributeAsKey('alias', false)
+                    ->prototype('array')
+                        ->children()
+                            ->arrayNode('servers')
+                                ->prototype('scalar')->end()
+                            ->end()
+                            ->arrayNode('events')
+                                ->cannotBeEmpty()
+                                ->useAttributeAsKey('tagName')
+                                ->prototype('array')
+                                    ->useAttributeAsKey('subTagName')
+                                    ->prototype('array')
+                                        ->useAttributeAsKey('eventName')
+                                        ->prototype('array')
+                                            ->children()
+                                                ->enumNode('type')->values(['counter', 'gauge', 'histogram_linear', 'histogram_exponential', 'histogram_user_specified'])->end()
```

Comment:
No timer ?

Reply:
I've listed here the type of graphs prometheus can handle. I think `timer` can ne represented in one of this chart type, the changing thing is it need to use `getTiming` on event instead of `getValue` to get data.
What if I add in config
```
->scalarNode('value')->default('value')
```